### ISSUE TITLE
[1.10.2] Undecorated chunks in RTG worlds

### ISSUE THREAD

**Issue by whichonespink44** [SpongePowered/SpongeForge#1596]
- SpongeForge version: 1.10.2-2281-5.2.0-BETA-2401
- Forge version: 1.10.2-12.18.3.2281
- Java version: 8
- Operating System: Windows
- Plugins/Mods:
  - [RTG-1.10.2-4.1.2.4](https://minecraft.curseforge.com/projects/realistic-terrain-generation/files/2381592)

**What is the expected result?**
Decorated chunks in all biomes.

**What is the current result?**
Undecorated chunks in some biomes (mainly Plains and Savanna).

[fml-server-latest.log](https://gist.github.com/whichonespink44/71684cd330d5db79dfd7978a6277d265)

---------------------

This issue is really noticeable in any Plains or Savanna biome, and occasionally noticeable in Forest and Taiga biomes.

https://github.com/Team-RTG/Realistic-Terrain-Generation/issues/1177

![](https://cloud.githubusercontent.com/assets/10927161/26425433/a4c2e936-40cc-11e7-9e37-8ea5cbd6d69a.png)

![](https://cloud.githubusercontent.com/assets/10927161/26425437/a8a0b614-40cc-11e7-923b-3bd1e7fcf79b.png)

**Comment by whichonespink44** [comment]
Hiya... just wondering if any progress has been made on this as we're about to release a big update to RTG and would love to be SpongeForge-compatible on launch if possible.

If there's anything we can do to help out, please let me know.

**Comment by bloodmc** [comment]
@Deamon5550 you able to take a look at this?

**Comment by merjilin** [comment]
I have just tested this on a brand new world with spongeforge not installed along with no additional mods, and undecorated chunks are still prevalent. 
Used: 
Forge: forge-1.10.2-12.18.3.2422-universal.jar
RTG: RTG-1.10.2-4.2.0.0-snapshot-5.jar

**Comment by Deamon5550** [comment]
@merjilin tell that to the mod developer then so they can look into it properly.

**Comment by srs-bsns** [comment]
@Zidane Why did you close this? This issue has not been dealt with yet.

This issue was opened in relation to the current stable release of RTG for 1.10.2 and the decoration issue has only ever been seen when used with SpongeForge.
merjilin is posting about an unstable alpha snapshot and it, likely, has no connection to this issue.

**Comment by merjilin** [comment]
I will do testing with the build shown in this issue and get back to you on that one, just to confirm everything.

**Comment by vectrixdevelops** [comment]
@Deamon5550 @Zidane The author made this issue.

**Comment by gabizou** [comment]
Not sure why this was closed @Deamon5550 considering one of the devs did open this issue. I'll find some time to work with them to review the issue again and probably figure something out.

**Comment by Deamon5550** [comment]
@gabizou Why the fuck are you bitching at me, I didn't close it? all I did was say if its reproducible without sponge then its probably not a sponge issue and the mod issue should be poked with that information...

**Comment by gabizou** [comment]
I misread the notification when the issue was closed, misread in that I understood that daemon closed the issue. I’m not bitching at anyone, just wondered why it was closed. Eventually I can get to this issue, just can’t guarantee it to be reached at in the immediate future.

**Comment by Deamon5550** [comment]
It was closed because it was stated that it is reproducible without sponge ergo its a problem with the mod not sponge.

**Comment by merjilin** [comment]
I have just tested it using the version specified in the issue, and that does have issues when it's with spongeforge. However, the latest snapshot available on curse does have undecorated chunks without spongeforge when I tested it, so I'm not sure what is happening there. As srs-bsns stated, it could be down to the snapshot being unstable. Sorry for the confusion. If the issue persists on the next stable release from RTG, I will open an issue on their github.
